xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -715,6 +715,11 @@ extension.oldflavors=$oldFlavor.trim()
 #-# Once the feature stabilizes, it will be enabled by default and the option to disabled it will be removed.
 # watchlist.realtime.enabled = false
 
+#-# [Since 7.1.5, 7.3.1, 7.4M3]
```

Comment:
7.4 is already out, and since this isn't a bugfix, it probably shouldn't be backported. Unless you can reformulate this as a bug instead of an improvement.